x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_preview_table.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiInMemoryTable } from '@elastic/eui';
+import type { RiskSeverity } from '../../../common/search_strategy';
+import { RiskScore } from '../../explore/components/risk_score/severity/common';
+
+import { HostDetailsLink, UserDetailsLink } from '../../common/components/links';
+import type { RiskScore as IRiskScore } from '../../../server/lib/risk_engine/types';
+import { RiskScoreEntity } from '../../../common/risk_engine/types';
+
+export const RiskScorePreviewTable = ({
+  items,
+  type,
+}: {
+  items: IRiskScore[];
+  type: RiskScoreEntity;
+}) => {
+  const columns = [
```

Comment:
is it worth to `useMemo` this array?

Reply:
Hey, why do you think we should do it here?

columns will recreate it we rerender RiskScorePreviewTable, which anyway will lead to the re-render of EuiInMemoryTable

Reply:
Yes, you are right, I was thinking about a situation where only the `type` is updated (and not the `items`), but that will never happen.
However, I think we can still improve this component by wrapping it in a `React.memo`, just to make sure the component only re-renders when it needs to.